app/src/main/java/org/p2p/wallet/sdk/facade/model/SolendMethodResultSuccess.kt

```diff
@@ -0,0 +1,63 @@
+package org.p2p.wallet.sdk.facade.model
+
+import com.google.gson.annotations.SerializedName
+import org.p2p.wallet.utils.Base58String
+import java.math.BigDecimal
+import java.math.BigInteger
+
+sealed interface SolendMethodResultSuccess {
```

Comment:
Why do we need sealed interface for all models?
Why we cannot set the result model we want for each method?

Reply:
in the background processing entity we can have only one when and handle result depending on type
that's how it was planned